Assets/Scripts/Connector/ValueInjector/RectTransformInjector.cs

```diff
@@ -0,0 +1,187 @@
+            {
+                rotation = value;
+                HasInjectRotation = true;
+            }
+        }
+        private Vector3 RotationEuler
+        {
+            get => rotationEuler;
+            set
+            {
+                rotationEuler = value;
+                HasInjectRotationEuler = true;
+            }
+        }
+        private Vector3 Scale
+        {
+            get => scale;
+            set
+            {
+                scale = value;
+                HasInjectScale = true;
+            }
+        }
+        private Vector2 AnchoredPosition
+        {
+            get => anchoredPosition;
+            set
+            {
+                anchoredPosition = value;
+                HasInjectAnchoredPosition = true;
+            }
+        }
+        private Vector3 AnchoredPosition3D
+        {
+            get => anchoredPosition3D;
+            set
+            {
+                anchoredPosition3D = value;
+                HasInjectAnchoredPosition3D = true;
+            }
+        }
+
+        [SerializeField] private GameObjectCollector baseGameObjectCollector = new GameObjectCollector();
+        [SerializeField] private StringCollector transformPathCollector = new StringCollector();
+        [SerializeField] private Vector3Collector positionCollector = new Vector3Collector();
+        [SerializeField] private QuaternionCollector rotationCollector = new QuaternionCollector();
+        [SerializeField] private Vector3Collector rotationEulerCollector = new Vector3Collector();
+        [SerializeField] private Vector3Collector scaleCollector = new Vector3Collector();
+        [SerializeField] private Vector2Collector anchoredPositionCollector = new Vector2Collector();
+        [SerializeField] private Vector3Collector anchoredPosition3DCollector = new Vector3Collector();
+
+        private GameObjectCollector BaseGameObjectCollector => baseGameObjectCollector;
+        private StringCollector TransformPathCollector => transformPathCollector;
+        private Vector3Collector PositionCollector => positionCollector;
+        private QuaternionCollector RotationCollector => rotationCollector;
+        private Vector3Collector RotationEulerCollector => rotationEulerCollector;
+        private Vector3Collector ScaleCollector => scaleCollector;
+        private Vector2Collector AnchoredPositionCollector => anchoredPositionCollector;
+        private Vector3Collector AnchoredPosition3DCollector => anchoredPosition3DCollector;
+
+
+        private bool HasInjectPosition { get; set; }
+        private bool HasInjectRotation { get; set; }
+        private bool HasInjectRotationEuler { get; set; }
+        private bool HasInjectScale { get; set; }
+        private bool HasInjectAnchoredPosition { get; set; }
+        private bool HasInjectAnchoredPosition3D { get; set; }
+
+        protected override void Inject()
+        {
+            var targetTransform = BaseGameObject.transform as RectTransform;
```

Comment:
Should be check if value is not `null`.

Reply:
Add `null` check for `targetTransform` at https://github.com/monry/UniFlow/pull/212/commits/0cb469583dad2ec2d24189777b9b2e570f7fc6de